jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/cpu/OverallCPULoadHandler.java

```diff
@@ -17,16 +17,23 @@
 public final class OverallCPULoadHandler implements RecordedEventHandler {
   private static final String METRIC_NAME_PROCESS = "process.runtime.jvm.cpu.utilization";
   private static final String METRIC_NAME_MACHINE = "process.runtime.jvm.system.cpu.utilization";
+  private static final String METRIC_NAME_MACHINE_MINUTE = "process.runtime.jvm.system.cpu.load_1m";
   private static final String METRIC_DESCRIPTION_PROCESS = "Recent CPU utilization for the process";
   private static final String METRIC_DESCRIPTION_MACHINE =
       "Recent CPU utilization for the whole system";
+  private static final String METRIC_DESCRIPTION_MACHINE_MINUTE =
+      "Average CPU load of the whole system for the last minute";
 
   private static final String EVENT_NAME = "jdk.CPULoad";
   private static final String JVM_USER = "jvmUser";
   private static final String JVM_SYSTEM = "jvmSystem";
   private static final String MACHINE_TOTAL = "machineTotal";
   private volatile double process = 0;
   private volatile double machine = 0;
+  private volatile double machineMinuteAverage = 0;
+  private static final int POLLING_INTERVAL_S = 1;
+  private int count = 0;
+  private double machineSum = 0;
```

Comment:
Does JFR guarantee that events are always dispatched on the same thread? If not, then this should probably be volatile as well.

Reply:
Good question. The _assumption_ is that the events are fired onto a SingleThreadExecutor but it is in theory possible that some could use a different model, although I would expect that the extra complexity would very much outweigh any possible throughput gain.

Reply:
I made it volatile and added synchronization